www/js/app.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+// 'weather' is referenced in index.html, 2nd arg is dependencies
```

Comment:
The code here looks good to me. I'm still learning Javascript and not as well versed in it as I would like to be, but it looks well maintained.